common/src/main/java/com/microsoft/identity/common/internal/net/HttpResponse.java

```diff
@@ -22,85 +22,35 @@
 // THE SOFTWARE.
 package com.microsoft.identity.common.internal.net;
 
-import androidx.annotation.NonNull;
-import androidx.annotation.Nullable;
-
-import net.jcip.annotations.Immutable;
-
 import java.util.Date;
 import java.util.List;
 import java.util.Map;
 
+import lombok.NonNull;
```

Comment:
in the HttpRequest.java we are keeping (not sure if they are used)
import androidx.annotation.NonNull;
import androidx.annotation.Nullable;

But i think we should be consistent?